src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,16 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article className={(todo.completed
+    ? 'TodoInfo TodoInfo--completed'
+    : 'TodoInfo')}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+    <UserInfo user={(!todo.user) ? null : todo.user} />
```

Comment:
it could be simplified. it would be better to check the user as well
```suggestion
    {todo.user && <UserInfo user={todo.user} />}
```